File: DeliveryBot/DeliveryBot.Db/Models/Address.cs
namespace DeliveryBot.Db.Models;

public class Address : Entity
{
    public string? AddressLine1 { get; set; }
    public string? AddressLine2 { get; set; }
    public string? AddressLine3 { get; set; }
    public string? AddressLine4 { get; set; }
    public string? TownCity { get; set; }
    public string? Region { get; set; }
    public string? Country { get; set; }
    public string? Postcode { get; set; }
}

File: DeliveryBot/deliverybot.client/src/interfaces/address.ts
export interface AddressDto {
  addressLine1?: string | undefined;
  addressLine2?: string | undefined;
  addressLine3?: string | undefined;
  addressLine4?: string | undefined;
  townCity?: string | undefined;
  region?: string | undefined;
  country?: string | undefined;
  postcode?: string | undefined;
}

export interface EditableAddressDto extends AddressDto {
  id?: string;
}